worldengine/draw.py

```diff
@@ -176,9 +177,9 @@ def draw_riversmap(world, target):
     sea_color = (255, 255, 255, 255)
     land_color = (0, 0, 0, 255)
 
-    for y in range(world.height):
+    for y in range(world.height):#TODO: numpy
         for x in range(world.width):
-            if world.ocean[y][x]:
+            if world.ocean[y, x]:
```

Comment:
The meat of this branch: Switching out [y][x] for [y, x]
I actually tested this and yes, the latter version is a bit faster. And with so many arrays in memory and such an incredible amount of calls to the operator[] I thought it might be worth doing this replacement to gain a little bit of speed overall (it amounted to ~1-2% in total).